CUDA images don't work in clang-cuda mode

I get this error
```
Run Build Command(s):/usr/bin/make -f Makefile cmTC_53c16/fast && /usr/bin/make  -f CMakeFiles/cmTC_53c16.dir/build.make CMakeFiles/cmTC_53c16.dir/build
make[1]: Entering directory '/root/gridtools/build/CMakeFiles/CMakeFiles/CMakeTmp'
Building CXX object CMakeFiles/cmTC_53c16.dir/TryClangCuda.cpp.o
/usr/bin/clang++-12  -isystem /usr/local/cuda/include -xcuda --cuda-path=/usr/local/cuda/bin/.. --cuda-gpu-arch=sm_60 -std=c++14 -MD -MT CMakeFiles/cmTC_53c16.dir/TryClangCuda.cpp.o -MF CMakeFiles/cmTC_53c16.dir/TryClangCuda.cpp.o.d -o CMakeFiles/cmTC_53c16.dir/TryClangCuda.cpp.o -c /root/gridtools/build/CMakeFiles/TryClangCuda.cpp
clang: warning: Unknown CUDA version. cuda.h: CUDA_VERSION=11050. Assuming the latest supported version 10.1 [-Wunknown-cuda-version]
In file included from <built-in>:1:
/usr/lib/llvm-12/lib/clang/12.0.1/include/__clang_cuda_runtime_wrapper.h:426:10: fatal error: 'curand_mtgp32_kernel.h' file not found
#include "curand_mtgp32_kernel.h"
```
Maybe cuda-minimal doesn't contain the library with this header...

--- a/cuda/Dockerfile
+++ b/cuda/Dockerfile
@@ -11,7 +11,7 @@ RUN apt-get update -qq && \
     rm -rf /var/lib/apt/lists/*
 
 RUN apt-get update -qq && \
-    DEBIAN_FRONTEND=noninteractive apt-get install -qq -y cuda-minimal-build-11-5 && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -qq -y cuda-minimal-build-11-5 libcurand-dev-11-5 && \
     rm -rf /var/lib/apt/lists/*
 
 ENV CUDACXX=/usr/local/cuda-11.3/bin/nvcc CUDAHOSTCXX=${CXX}

